tests/sku-codemods/sku-codemods.test.ts

```diff
@@ -0,0 +1,86 @@
+import { describe, it } from 'vitest';
+import fs from 'node:fs/promises';
+import { runSkuCodemod } from '@sku-private/test-utils';
+import dedent from 'dedent';
+import { createFixture } from 'fs-fixture';
+
+describe('sku codemods', () => {
+  describe('"transform-vite-loadable" codemod', async () => {
+    const filesToTest = [
```

Comment:
Any reason this can't be refactored to use [`describe.for`](https://vitest.dev/api/#describe-for) for better integration with the test runner?